photon-ml/src/main/scala/com/linkedin/photon/ml/avro/data/DataProcessingUtils.scala

```diff
@@ -84,6 +84,39 @@ object DataProcessingUtils {
     }
   }
 
+  /**
+    * Given a GenericRecord, build the random effect id map:
+    *     (random effect name -> random effect id value)
+    *
+    * @note Exposed for testing purpose.
+    * @param record the avro generic record
+    * @param randomEffectIdSet a set of random effect id names
+    * @return the random effect id map of (name -> value)
+    */
+  protected[avro] def makeRandomEffectIdMap(record: GenericRecord,
+      randomEffectIdSet: Set[String]): Map[String, String] = {
```

Comment:
Sorry, another minor one @timyitong: ```record: GenericRecord``` should be on its own line, aligned with ```randomEffectIdSet```.

I don't believe we have a scalaStyle rule that checks for this currently. I'll look into that today if I have time.